packages/directory/addon/templates/directory.hbs

```diff
@@ -22,6 +22,7 @@
   {{dir-sidebar classNames='directory__sidebar'}}
 
   <section class='directory__table'>
+    <h1>{{title}}</h1>
```

Comment:
Can we add a class for styling?

Reply:
It's already super easy to target for styling. if we have more than one h1 under .directory__table that's a problem.

Reply:
Had to add an outer div anyway due to  wierd behavior in ember-light table, so I dislike having bare <divs> so that has a class now.